### DISCUSSION TITLE
Can I add a new field (or fields) in sigma for my internal application?

### DISCUSSION THREAD

**Discussion by tpseers** [SigmaHQ/sigma-specification#67] [Q&A] [answered]
I need to track some false positive/negative events in my sigma rules, can I add a new field like "test" (similar to "references" field)? Or I can only use the standard SIGMA fields? I don't want to add those test event info under "references" field. 

In other words, is there any restriction for adding customised fields?

Thanks

**Comment by nasbench** [comment, marked as answer]
Hi,

You can add as many fields as you like for your internal needs. The SIGMA specification lists the fields that are currently supported and approved by the SigmaHQ team to work with different toolings such as PySIGMA or SIGMAC.

**Reply by tpseers** [reply]
Great! Thanks @nasbench for the quick clarification!

**Reply by nasbench** [reply]
No worries.
